pages/settings/settings.js

```diff
@@ -11,8 +11,12 @@ var userAgentCheckbox = document.getElementById('checkbox-user-agent')
 var userAgentInput = document.getElementById('input-user-agent')
 
 function showRestartRequiredBanner () {
-  banner.hidden = false
+  banner.hidden = false;
+  settings.set('restartNow', true)
 }
+settings.get('restartNow', (value) => {
+    if(value && value == true) showRestartRequiredBanner();
```

Comment:
Three style issues here:

* Space after "if" and === rather than == (standard should catch this)
* Use braces around the if statement (If you haven't read it, here's an [interesting and good example](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) of why you should do this)